manuscript/manuscript.md

```diff
@@ -217,6 +225,15 @@ spirit of this rule is that data should be as "pure" as possible when they are
 stored. If derivations occur, they should be documented by also archiving
 relevant code and intermediate datasets.
 
+A cryptographic hash (e.g., SHA or MD5) of the raw data should be generated and
```

Comment:
Is there a way to re-write this so that non-technical users can not only understand but also adopt the generation and distribution of hashes with their data?